packages/terra-avatar/docs/README.md

```diff
@@ -0,0 +1,32 @@
+# Terra Avatar
+
+The terra-avatar component displays an avatar which can be either an image or text. If neither an image or text is provided, a variant-based fallback icon is used. The avatar displays in a circular wrapper.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-avatar`
+
+## Usage
+
+```jsx
+import React from 'react';
+import Avatar from 'terra-avatar';
+
+<Avatar
+  image="https://path/to/image.jpg"
+  alt="placeholder"
+  variant="user" 
+/>
+```
+
+## Component Features
+
+<!-- Uncomment supported features.
```

Comment:
You can uncomment any supported features this component has. This helps users better understand what the component offers.

Reply:
Updated in this [commit](https://github.com/cerner/terra-core/commit/63281dd2f99a1f829acc97efee1548a149368b78).